core/client/app/components/gh-uploader.js

```diff
@@ -28,14 +28,14 @@ var PostImageUploader = Ember.Component.extend({
         $this.on('imagecleared', function () {
             self.sendAction('canceled');
         });
-    }.on('didInsertElement'),
+    },
 
-    removeListeners: function () {
+    willDestroyElement: function () {
```

Comment:
The one thing that I think was better in the prototype extension versions was that the methods were self documenting by their name. Before, I could just see that this was called `removeListeners` and know what it did. @acburdine if you've the time, I think it would be worth naming the anonymous functions with their old names, ie
```js
removeListeners: function () {}.on('willDestroyElement')
//to
willDestroyElement: function removeListeners() {}
```

Because yay readability